### ISSUE TITLE
[Feature request] Remove song from queue

### ISSUE THREAD

**Issue by CodeWithMa** [leinelissen/jellyfin-audio-player#164]
Currently I can only click on "Clear Queue" to delete the complete queue.

It would be nice to remove single songs from the queue by for example swiping them to the side or with a long press and a context menu item "Remove from queue".

**Comment by leinelissen** [comment]
Fair enough! As soon as I find a library that does the swipey thing well, I'll try and put this in.

**Comment by CodeWithMa** [comment]
Did some searching and found this: 

- https://github.com/hammerjs/hammer.js (Seems to be deprecated)
- https://github.com/biodiv/contactjs

**Comment by leinelissen** [comment]
Most probably it's something like this: https://github.com/jemise111/react-native-swipe-list-view, though there haven't been updates in a while and I need to check whether it uses native primitives.

**Comment by maltfield** [comment]
@leinelissen what I would recommend is to make a "slide" gesture, such that if someone taps a given row of the queue (where a row is a song in the queue) and then swipes far left or far right, then the song is removed.

And, at the same time, a small toast message + button combo appears at the bottom of the screen that says "song removed" the button next to this message says "undo"

This is how newpipe does it:

<img src="https://user-images.githubusercontent.com/85039990/166748260-27521847-8eb2-44ec-b4f6-7660f3103afb.png" />

I think this UX is extremely intuitive

**Comment by maltfield** [comment]
See also this request to be able to change the order of songs in the queue https://github.com/leinelissen/jellyfin-audio-player/issues/175
